unicodetools/src/main/java/org/unicode/xml/GeneratePropertyValues.java

```diff
@@ -154,8 +154,19 @@ public static void main(String[] args) throws Exception {
                             "Could not find or create " + options[OUTPUTFOLDER].value);
                 }
             } else {
-                throw new IllegalArgumentException(
-                        "Missing command line option: --outputfolder (or -f)");
+                try {
+                    destinationFolder = new File(Settings.Output.GEN_DIR + "uax42\\fragments\\");
+                    if (!destinationFolder.exists()) {
```

Comment:
duplicate code -- I would try to get the destinationFolder into a variable whether it's from the command line or the default, and then use the same code to check exists(), call mkdirs(), etc.

Reply:
Fixed. And the original was pretty lazy on my part...